app/controllers/announcements_controller.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+class AnnouncementsController < ApplicationController
+  def index
+    @announcements = Announcement.all
+  end
+
+  # show
+  def show
+    @announcement = Announcement.find params[:id]
+  end
+
+  # new
+  def create
+    announcement = Announcement.new announcement_new_params
+    announcement.date = DateTime.now.getlocal
+
+    redirect_to announcements_path if announcement.save
+  end
+
+  # edit
+  def update
+    announcement = Announcement.find params[:id]
+
+    redirect_to announcements_path if announcement.update announcement_edit_params
```

Comment:
If I remember correctly `announcement.update` throws an error instead of returning true/false

Reply:
How do I go about checking what `announcement.update` returns?

Reply:
I don't get what you mean...? Haha. If the update function works fine in the view, does that means all is well?

Reply:
Oh, it returns true. My original question was actually how to check if `announcement.update` returns true/false because I can't seem to do a `System.out.println` equivalent in the controller.